apps/elewa-group-website/src/app/app.component.html

```diff
@@ -1 +1,4 @@
-<elewa-group-home-page></elewa-group-home-page>
\ No newline at end of file
+
+
+ <elewa-group-job-section></elewa-group-job-section
```

Comment:
Job section needs to be on the home page.
only home page tag should be on thing file..